project 1/styles.css

```diff
@@ -0,0 +1,70 @@
+body {
+    background-color: #DCE6D5;
+}
+
+
+h1 {
+    color: #223809;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 45px;
+    font-weight: 600;
+    line-height: 20px;
+    letter-spacing: .3px;
+    text-align: center;
+}
+
+h2 {
+    color: #223809;
+    font-family: 'Libre Baskerville', serif;
+    font-size: 25px;
+    font-weight: 400;
+    line-height: 15px;
+    letter-spacing: .3px;
+    text-align: center;
+}
+
+p {
+    color: #424242;
+    font-family: 'Libre Baskerville', serif;
+    line-height: 35px;
+    font-size: 15px;
+    font-weight: 500;
+    letter-spacing: .2px;
+    text-align: left;
+}
+
+blockquote {
+    color: #223809;
+    font-family: 'Libre Baskerville', serif;
+    line-height: 40px;
+    font-size: 30px;
+    letter-spacing: .3px;
+    text-align: center;
+}
+
+* {
+    box-sizing: border-box
+}
+
+.body-text {
+    background-color: #F9F9F4;
+    border: 5px solid #223809;
+    height: 2000px;
```

Comment:
Without a `height` rule, this `div` will stretch to fit your text, which is ideal. I recommend removing the `height` rule and instead making sure the spacing is correct via padding.